sample/ws-chat-server.c

```diff
@@ -255,4 +255,9 @@ main(int argc, char **argv)
 	libevent_global_shutdown();
 
 	return 0;
+
+cleanup:
```

Comment:
Can you please also add a check `http_server` and don't forget to clean the `sig_int` here